### PR TITLE
Added installation instructions for Vim + Vim-LSP

### PR DESCRIPTION
**Pull request by jHwls**
Thank you for your work on this library! Had a go at adding Vim-specific instructions, referencing this [issue](https://github.com/lexical-lsp/lexical/issues/228)

### CODE REVIEW

**Comment by scohen** [comment]
@jHwls Fix the suggestion, then you're good! I have no idea if this works or not, but YOLO 🤣 .

**Comment by jHwls** [comment]
> @jHwls Fix the suggestion, then you're good! I have no idea if this works or not, but YOLO 🤣 .

🎉 

What suggestion @scohen ?

Realizing I should replace minpac with the native vim Package manager so this is a bit more general. I'll do that also

**Comment by jHwls** [comment]
Btw @scohen works great! The as-you-type compilation is really nice. Some issues right on startup with formatting/format on save I need to dig into and was hoping to ask about on the discord shortly, but...

Completion:
Screenshot 2023-08-10 at 2 48 43 PM

Linting, via ALE:
Screenshot 2023-08-10 at 2 49 11 PM

LSP diagnostics list:
Screenshot 2023-08-10 at 2 49 50 PM


👍

File: pages/installation.md [hunk 1]
```diff
@@ -138,3 +139,41 @@ In version `>= 0.9.0`, the key is to append the custom LS configuration to [lspc
...
 If the configuration above doesn't work for you, please try this minimal [neovim configuration](https://github.com/scottming/nvim-mini-for-lexical), It can eliminate other plugin factors.
+
+### Vim + Vim-LSP
+
+An example of configuring Lexical as the Elixir language server for 
+[Vim-LSP](https://github.com/prabirshrestha/vim-lsp). Uses the newer vim9script syntax but 
+can be converted to Vim 8 etc (`:h vim9script`).
+
+```
+vim9script
```
[the remark this answers is not in the document]
Comment (jHwls) [reply pages/installation.md:150]: Ah, I see! Yah, it's not actually the md language block indicator. It's a statement required at the top of the file while vim migrates to new syntax:

> To recognize a file that can be imported the `vim9script` statement must appear as the first statement in the file...

Figured I'd include it here to flag up that it's in the newer vim 9 syntax, but if you think it's confusing I could remove it?

[the remark this answers is not in the document]
Comment (scohen) [reply pages/installation.md:150]: omg. sorry about that. Maybe it should be a comment?

[the remark this answers is not in the document]
Comment (scohen) [reply pages/installation.md:150]: Alright, approved.

**Review by scohen — APPROVED** [review]

**Pull request merged**